modules/ROOT/pages/upgradestudio.adoc

```diff
@@ -22,3 +22,36 @@ How to upgrade your existing Studio to a newer version.
 Maintenance versions are cross-compatible, meaning that it is not required that your Studio version matches the target runtime version.
 
 For example, your production runtime can be in 2021.1-0408 and your Studio in 2021.1-0218
+
+== Upgrading your project to the project composition
+
+Version 2021.2 introduces major changes to the way the Studio manages projects. When importing or cloning your project in this version or any later version, it will be transformed into a Maven project. All your project dependencies (connectors, libraries, actor filters...) will be managed with Maven.
+
+*_BOS Import_*
+
+In the BOS import wizard, an additional step has been added. The Studio is going to scan your project dependencies and try to find them on the configured Maven repositories (by default Maven Central is configured).
+
+If the dependencies are found then they will be used as remote dependencies (not committed in your project) otherwise they will be added to the project local repository store and will be committed to your project. 
```

Comment:
```suggestion
If the dependencies are found then they will be used as remote dependencies (not copied in your project) otherwise they will be copied in the project and will be committed with your project sources. 
```